### PR TITLE
Added Add and remove functionality with JS Objects

### PR DESCRIPTION
**Pull request by Hector096**
Milestone1: add JS object with add and remove the item methods
Milestone2: persist data to localstorage
Milestone3: add DOM manipulation to HTML

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    title: title.value,
+    author: author.value,
+  });
+  localStorage.setItem('books', JSON.stringify(books));
+  title.value = '';
+  author.value = '';
+  setItems();
+});
+
+setItems();
```
Comment (arslanbisharat) [inline app.js]: - [ ] I cannot add any book. Please make sure the function is working properly. 
Capture

- [ ] Please add a proper function to remove a book. 
Capture

- [ ] Please make sure the data is preserved in the browser's memory by using localStorage.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+  </ul>
+  <form action="#">
+    <input type="text" placeholder="tilte" id="title"/>
+    <input type="text" placeholder="author" id="author"/>
+    <button type="button" id="add-btn">
+      Add book
+    </button>
+  </form>
+</html>
```
Comment (arslanbisharat) [inline index.html:23]: - [ ] Please add the remove button in UI. It is a basic requirement.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: app.js (hunk 1 above)
Comment (henatan99) [reply app.js]: This issue is addressed well. Excellent work! :clap:

File: index.html (hunk 2 above)
Comment (henatan99) [reply index.html:23]: This issue is resolved. Nice job! :clap: :+1:

**Review by henatan99 — APPROVED** [review]
Hi @Hector096 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**